voluncheer/opportunityboard/tests_views.py

```diff
@@ -104,3 +107,83 @@ def test_search_by_categories(self):
         filters.subcategory = None
         filters.subsubcategory = self.subsubcategory
         self.assertEqual(filters.search().count(), 1)
+
+    def test_search_by_duration_lte2hrs(self):
```

Comment:
These tests are quite difficult to read. It would be a bit easier if the event started at 0:00 and then you set the durations based on incrementing from there. It's not realistic from and actual opportunity point of view but the logic will still apply and when someone reads it they will know that the duration is 1.5 hours when the end time is 1:30. Thoughts?

The tests seem logically sound, just hard to read.

You might also want to look into creating "subTest"s within each test function to differentiate and annotate which durations you are testing and which results you are expecting: https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest

Something like: `with self.subTest("one_hour_exists"):` etc.

Reply:
Followed your advice on the 0:00 start time for the test opportunity. Please let me know what you think.

Reply:
Might need to push a commit? I don't see any difference.

Reply:
Make sure you are viewing the changes from all commits. I noticed the same experience you had, the diffs didn't contain my latest updates. Might be a bug in GitHub

Reply:
Now I see it. Looks good